### PR TITLE
Api version 1.1, now with more resize

### PR DESCRIPTION
**Pull request by carolynvs**
cc @everett-toews

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-3a176ad140/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-cb286978cb/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-30974884b0/).

**Comment by carolynvs** [comment]
I believe that the links aren't working (I've tried them both ways with getcarina.com, and relative links) is that nexus is replacing the root domain. I think the links will look fine once we merge and it's live on getcarina.com instead of staging.

File: _reference/2015-10-17-release-notes.md [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
 description: Notes on Carina releases
 topics:
   - docker
 ---
+
+### December 13, 2016
+
+#### What's new
+
+* Resize next generation Carina clusters using the website or [the CLI](/betadocs/docs/reference/carina-cli/#resize-a-cluster).
```
Comment (everett-toews) [inline _reference/2015-10-17-release-notes.md]: s/website/UI/ ?

And turn it into a link please.

Comment (carolynvs) [inline _reference/2015-10-17-release-notes.md]: Do you want to link to ui.getcarina.com? We don't have user facing doc on resize, other than the cli doc for the command.

Comment (everett-toews) [inline _reference/2015-10-17-release-notes.md]: Yep

**Pull request closed without merging**